superflore/generators/bitbake/gen_packages.py

```diff
@@ -35,8 +35,12 @@
 
 
 def regenerate_installer(
-    overlay, pkg, distro, preserve_existing, tar_dir, md5_cache, sha256_cache
+    overlay, pkg, distro, preserve_existing, tar_dir, md5_cache, sha256_cache,
+    skip_keys
 ):
+    if pkg in skip_keys:
+        warn("Package '%s' is on skip-keys, skipping..." % pkg)
```

Comment:
I'm not a huge fan of the phrasing here. Maybe something like `"Package '%s' is in skip-keys list, skipping..."`? I'm not sure.

Reply:
Done, thanks.